packages/cf-component-toast/src/index.js

```diff
@@ -0,0 +1,8 @@
+import { applyTheme } from 'cf-style-container';
+
+import ToastUnstyled from './Toast';
+import ToastTheme from './ToastTheme';
+
+const Toast = applyTheme(ToastUnstyled, ToastTheme);
+
+export { Toast as default, ToastUnstyled, ToastTheme };
```

Comment:
please change this to:

```js
export { Toast, ToastUnstyled, ToastTheme };
```

We want to be consistent across the library with migrated components.